From fe8794a5e1fcca60982b43ac9ab7617a5a213694 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Mon, 30 Nov 2020 20:17:52 +0200
Subject: [PATCH] [SW-20154] goya: add header header guard for common.h

Add a header guard to prevent multiple inclusions.
There is a checkpatch warning as shown below for
the license header needs to be there on
1st line. Hence, changing the header too.

WARNING: Misplaced SPDX-License-Identifier tag - use line 1 instead

Change-Id: Iae0438c7b6a126dd92be4a967782f206a03bcd7e
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/goya/common.h b/goya/common.h
index 5bfcc1e..6a64674 100644
--- a/goya/common.h
+++ b/goya/common.h
@@ -1,9 +1,11 @@
-/*
- * (C) Copyright 2019 HabanaLabs, Ltd.
- * Noam Camus <noamikong@gmail.com>
+/* SPDX-License-Identifier: GPL-2.0+
  *
- * SPDX-License-Identifier:     GPL-2.0+
+ * (C) Copyright 2019-2020 HabanaLabs, Ltd.
+ * All Rights Reserved.
  */
+
+#ifndef COMMON_GOYA_H_
+#define COMMON_GOYA_H_
 
 #include <preboot.h>
 
@@ -18,3 +20,5 @@
 #define FUSE_WORDS_PER_BANK	0x10
 #define I2C_SLAVE_ADDRESS	0x4f
 #define I2C_SLAVE_ADDRESS_DBG	0x47
+
+#endif /* COMMON_GOYA_H_ */
